recipes/pytrilinos/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "PyTrilinos" %}
+{% set version = "12.10.1" %}
+{% set sha256 = "ea8ff573bb1f452f2585950cdabe7e2c84cb35013844c58dc878c66bd4245e06" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: trilinos-{{ version }}-Source.tar.gz
+  url: http://trilinos.csbsju.edu/download/files/trilinos-{{ version }}-Source.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patch.txt
```

Comment:
Can you change the file extension to `.patch`? Also can you explain in a comment above why this patch is needed?